app/src/main/java/com/quran/labs/androidquran/ui/util/TranslationsSpinnerAdapter.java

```diff
@@ -36,6 +37,9 @@ public TranslationsSpinnerAdapter(Context context,
     // and the String[] constructor makes a new (immutable) List with the items of the array.
     super(context, resource, new ArrayList<>());
     this.layoutInflater = LayoutInflater.from(context);
+    List<String> transList = new ArrayList<String>(Arrays.asList(translationNames));
```

Comment:
I tried this, getting `java.lang.UnsupportedOperationException` on the next line however when trying to add to the list. Is there another way you intended to do this? jzk